doc/users/next_whats_new/per_subplot_mosaic.rst

```diff
@@ -0,0 +1,22 @@
+Per-subplot keyword arguments  in ``subplot_mosaic``
+----------------------------------------------------
+
+It is now possible to pass keyword arguments through to the creation to each
+specific call to ``add_subplot`` in `.Figure.subplot_mosaic` and
+`.pyplot.subplot_mosaic` :
+
+.. plot::
+   :include-source: true
+
+   fig, axd = plt.subplot_mosaic(
+       "AB;CD",
+       per_subplot_kw={
+           "A": {"projection": "polar"},
+           ("C", "D"): {"xlabel": "X Label"},
```

Comment:
```suggestion
           ("C", "D"): {"yscale": "log"},
```

feels a bit more safe. While one can define a lot of properties here, I'm hesitant to encourage using this for configuration of the whole plot. In particular labels are usually so individual that I'm not sure I want to see them in here (but I may change my opinion on that). The scale is a more fundamental property more similar to projections.

Reply:
ok, I'll change it here and in the tutorial.